src/App/HelpModal.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+  import Modal from "@app/components/Modal.svelte";
+</script>
+
+<style>
+  .hotkeys {
+    gap: 3rem;
+    align-items: flex-start;
+    justify-content: center;
+    display: flex;
+    color: var(--color-foreground-6);
+  }
+
+  .key {
+    border: 1px solid var(--color-secondary-5);
+    box-shadow: inset 0 -4px 0 var(--color-secondary-5);
+    height: 36px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: var(--border-radius-small);
+    background-color: var(--color-secondary-1);
+    min-width: 2rem;
+    padding: 0 1rem 4px 1rem;
+  }
+
+  .description {
+    text-align: left;
+  }
+
+  .pair {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .group {
+    display: flex;
+    gap: 2rem;
+    flex-direction: column;
+  }
+</style>
+
+<Modal emoji="⌨️" title="Keyboard shortcuts" showCloseAction={false}>
+  <div slot="body">
+    <div class="hotkeys">
+      <div class="group">
+        <div class="pair">
+          <div class="key txt-bold">?</div>
+          <div class="description">Shortcuts</div>
+        </div>
+
+        <div class="pair">
+          <div class="key txt-bold">/</div>
+          <div class="description">Search</div>
+        </div>
+
+        <div class="pair">
+          <div class="key txt-bold">tab</div>
+          <div class="description">Navigate</div>
+        </div>
+      </div>
```

Comment:
Navigating via <kbd>Tab</kbd> actually works, but for some reason we have disabled the outline highlighting. If we re-enable it via:
```diff
diff --git a/public/index.css b/public/index.css
index 4227b2e0..0fdd0137 100644
--- a/public/index.css
+++ b/public/index.css
@@ -1,5 +1,4 @@
 * {
-  outline: none !important;
   box-sizing: border-box;
 }
```

Then it shows up:

https://user-images.githubusercontent.com/158411/214249816-5a7857ce-9a60-4358-8bf6-5f2508f173e1.mov

Wonder if we should re-enable the highlighting for people with different vision.

Reply:
Yeah the outlining would make tab navigation more explicit, if I remember correctly we did surpress it, since it also shows up at some points when mouse clicking or keyboard input.
Your video shows that the header is traversed correctly, but if you want to click on a pinned project you won't get to it with tabbing.
I think the correct way is to add `focus` and `focus-within` css selectors and add tab indexes where necessary. But this should be a separate PR

Reply:
d90e655ad9b94b523e2084078242624c8e39c444

Reply:
👍